requirements.txt

```diff
@@ -2,5 +2,5 @@ numpy
 pyzmq
 zerorpc
 spacy
-https://github.com/explosion/spacy-models/releases/download/en_core_web_sm-2.0.0/en_core_web_sm-2.0.0.tar.gz#egg=en_core_web_sm
+https://github.com/explosion/spacy-models/releases/download/en_core_web_lg-2.1.0a0/en_core_web_lg-2.1.0a0.tar.gz#egg=en_core_web_lg
```

Comment:
Noting: we should try https://github.com/explosion/spacy-models/releases/download/en_core_web_lg-2.0.0/en_core_web_lg-2.0.0.tar.gz#egg=en_core_web_lg==2.0.0 for this

(as you made a change in the Spacy module - we have to expose it in requirements.txt -- so that its reflected in everyone's machine Cc @tzunun)